samples/SampleWebApp/Web.config

```diff
@@ -32,6 +32,10 @@
     <add key="jsonConnectionString1" value="WILL be replaced by 'SimpleJson' in 'Flat' jsonMode." />
     <add key="jsonConnectionString2" value="WILL NOT be replaced by 'SimpleJson' in 'Sectional' jsonMode." />
     <add key="jsonCustomSetting1" value="Will be replaced by 'SimpleJson' with prefix='customSettings:' and stripPrefix='true'." />
+    <add key="SecretFromFile1" value="This will be replaced by KeyPerFile." />
+    <add key="SECRETFROMFILE2" value="As will this one." />
+    <add key="SubFeature--FeatureSecretA" value="This will be replaced by KeyPerFile if keyDelimiter is set to '--'." />
```

Comment:
Could you please more specific and point out as File1?  This could be confusing and leading to confusion